### ISSUE TITLE
Prevent Alsa snd-aloop to become primary sound card

### ISSUE THREAD

**Issue by thetotalchaos** [falkTX/Cadence#187]
There is an issue i am experiencing, when i install and use Cadence to control the Jack server. 
It creates a virtual alsa-loop device, which often claims to be the default audio device. 
My solution (that i think should be somehow implemented by default), is to make a script that makes my hardware soundcard to be my top device.
So i create a file: /etc/modprobe.d/alsa-base.conf that contains this lines. 
`options snd_aloop index=-2
options snd_usb_audio index=-2`

The devices may vary depending of your hardware, but the idea is that i move the alsa-loop and usb-audio (my MIDI Keyboard) at the bottom of the audio devices list.

**Comment by falkTX** [comment]
I dont think this is something cadence has to do, but on kxstudio repos (using kxstudio-default-settings package) it enables snd-aloop. I have been patching the alsa lib package to get this "not primary soundcard" stuff, but having it on a separate file is a good idea, I forgot about it... (I guess it was mostly not knowing if it was possible or not)

Best thing cadence can do is add a check for this case, and warn the user and inform of possible solutions.

**Comment by thetotalchaos** [comment]
Keep in mind that there will be an use case (like in my Arch System), when Cadence is used apart from kxstudio-settings and patched packages.
